uk_election_ids/election_ids.py

```diff
@@ -409,3 +367,57 @@ def __repr__(self):
 
     def __eq__(self, other):
         return type(other) == IdBuilder and self.__dict__ == other.__dict__
+
+    @classmethod
+    def parse_from_id(cls, election_id: str) -> "IdBuilder":
+        """
+        Parses a string in to an IdBuilder object
+
+        """
```

Comment:
I've not attempted to compile the docs - you'll have to check this, but you might need an additional plugin like https://github.com/tox-dev/sphinx-autodoc-typehints if you want sphinx to use the typehints instead of types from the docstring (I should probably know this). On the topic of docs, you might also consider

- adding a `Raises: ...` to the docstring, to document what exceptions may be thrown
- adding a usage example to the docs: https://github.com/DemocracyClub/uk-election-ids/blob/master/docs/api.rst

Reply:
Just so you know, without any additional plugins: 
![image](https://user-images.githubusercontent.com/242329/207717767-e7487f28-e3e6-40f2-8c27-f6c8cf91376f.png)

So yeah, it shows the signature but doesn't expand it into the `dl` we see on other methods with doc strings.